src/test/java/com/josdem/vetlog/controller/UserControllerTest.java

```diff
@@ -72,9 +74,21 @@ void shouldShowCreateUserForm(TestInfo testInfo) throws Exception {
     }
 
     @Test
```

Comment:
In this test "not saving user due to invalid email" `@Transactional` is not necessary since it is a negative scenario (We will not save an user because the email in not valid).
Please remove:
- `@Transactional` annotation
-  Set up data before the test code

Reply:
That's certainly true.
I'll fix it!

Reply:
@josdem 
I tried to fix it
Maybe this is okay?